src/Creatable.js

```diff
@@ -151,6 +151,14 @@ const Creatable = React.createClass({
 		});
 	},
 
+	componentWillUpdate (nextProps, nextState) {
+		// Tricky: Async clears the select inputValue when deleting
+		// an option, but it doesn't fire onInputChange.
```

Comment:
Ahh - I'll update the comment. The other place we keep `this.inputValue` in-sync is in an onInputChange handler.